extern/bundled/python-virtualenv/install.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -euo pipefail -x
+
+unset PYTHONPATH  # existing PYTHONPATH can interfere
+
+( # install a Python virtualenv
+if ! type virtualenv &>/dev/null; then
+    # if virtualenv isn't available, install it locally along with pip
+    [[ -e bootstrap/virtualenv.py ]] || {
+        curl -RLO https://bootstrap.pypa.io/get-pip.py &&
+        python get-pip.py virtualenv --ignore-installed --target bootstrap
```

Comment:
Agree that it's probably a bad idea to pollute the runtime user's env. For internal use, should we just stick with `python3` (better unicode handling at least, also no ambiguity)? Do we ever need to bundle ddlib (assuming it's the ancient feature lib)?